MapleServer2/Types/Player.cs

```diff
@@ -198,8 +199,8 @@ public static Player NewCharacter(byte gender, /*Job jobType*/ int job, string n
             {
                 SkillTabs = skillTabs,
                 AccountId = 0x1111111111111111,
-                CharacterId = BitConverter.ToInt64(Guid.NewGuid().ToByteArray(), 0),
-                CreationTime = DateTimeOffset.UtcNow.ToUnixTimeSeconds(),
+                CharacterId = GuidGenerator.Long(),
+                CreationTime = DateTimeOffset.UtcNow.ToUnixTimeSeconds() + AccountStorage.TickCount,
```

Comment:
the same as above in the same class, could use session.ServerTick ?

Reply:
`Player` does not have access to `GameSession` because it is a Type so for the 2 hardcoded characters it cannot use a `GameSession` property, furthermore we want the `TickCount` from when the client first connected, the `session.ServerTick` updates over time.